pages/linux/prt-get.md

```diff
@@ -1,6 +1,6 @@
 # prt-get
 
-> The advanced CRUX package manager.
+> A CRUX package manager.
```

Comment:
This implies that there is more than 1 package manager. Is this the case? If not, we should keep using `The` instead.

Reply:
I've done some research and didn't find any other package manager.
I'll wait for a few hours to give others some time to read this (maybe someone knows if there is another package manager) and then I'll change it.